clang/tools/clang-format/ClangFormat.cpp

```diff
@@ -701,9 +711,8 @@ int main(int argc, const char **argv) {
   }
 
   if (FileNames.empty()) {
-    if (isIgnored(AssumeFileName))
-      return 0;
-    return clang::format::format("-", FailOnIncompleteFormat);
+    return clang::format::format("-", FailOnIncompleteFormat,
+                                 isIgnored(AssumeFileName));
```

Comment:
```suggestion
    if (isIgnored(AssumeFileName)) {
      outs() << MemoryBuffer::getSTDIN()->get()->getBuffer();
      return 0;
    }
    return clang::format::format("-", FailOnIncompleteFormat);
```
And leave the rest of the function alone.